guardian/story-packages
# Bump Play to 2.8

~**NB:** Depends on https://github.com/guardian/editorial-tools-platform/pull/646. Don't merge until this is in place – the app will not start!~ Now sorted.

## What does this change?

Bumps Play to 2.8, and pan-domain-auth to 1.2.0.

Skips Play 2.6, as it requires a different pan-domain-auth setup.

## How to test

Run locally, or in CODE. Browse, create, and modify story packages. Does everything work as expected?

~NB: Because we've recreated the story-packages stack recently, the old packages currently served by CAPI do not work! Additionally, because (I think!) the story-packages stream is not being read by CODE CAPI, new updates cannot be searched. So the only functionality we can currently test is creating a new story-package and editing that.~

~So at the moment, we can prove that~
 - ~we can talk to CAPI~
 - ~we can read and write packages from DynamoDB~
 - ~we can write updates on a stream (I've tested this locally)~

~But we cannot verify the search-for-package / select-package behaviour.~

CAPI's stream-reading behaviour is now sorted. I've made one behavioural change in [1500599](https://github.com/guardian/story-packages/pull/174/commits/15005996b7c0da5e0ed6baab27f2518115c74562) to make sure that the app does not 500 when CAPI surfaces content that is not in story-package's DB – instead, we just filter out the packages that we don't recognise. That seems less disruptive for the user – we still log the error message as normal. As a result, search should now work as well.
